src/api/model.js

```diff
@@ -34,4 +34,16 @@ app.post('/', (req, res) => {
         });
 });
 
+app.delete('/', (req, res) => {
+    addAction('DELETE_MODEL', {
+        modelAddress: req.query.modelAddress
+    }).then(() => {
+        res.successJson({
+            models: ModelStore.getModels()
```

Comment:
I don't think we should return all models; I think returning the deleted model would be better. Becomes more in-line with the typical remove operation for most data structures. Plus it is slower.